package.json

```diff
@@ -3,22 +3,69 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/core": "7.2.2",
+    "@svgr/webpack": "4.1.0",
```

Comment:
Use the exact package version.

That way we always know what dependency we are using. If a dependency version has the ^ caret, during npm install, npm will install the version of the package with major changes.